lib/index.js

```diff
@@ -1,24 +1,27 @@
+var _ = require('lodash');
+
+
 module.exports = function(server, options, next) {
-  
+
   if(!options.redirects) {
     options.redirects = {};
   }
-
   var log = options.log;
   var log404 = options.log404;
-
   var routes = [];
 
   var redirector = function(request, reply) {
     var routePath = request.route.path;
     var redirectTo = options.redirects[routePath];
+    _.each(request.params, function(param, paramName){
```

Comment:
nevermind, this only gets called for the route that is registered, not every request